music_album.rb

```diff
@@ -0,0 +1,10 @@
+require './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify, **args)
+    super(**args)
+    @on_spotify = on_spotify
+  end
```

Comment:
It is best if you put the exact quantity of arguments, using `**args` is bad for class creation. Theoretically, you can infinite arguments, however the `Item` class only allows for like 4 to 5. so if you put 6 or more the program will crash.
```suggestion
  def initialize(on_spotify, publish_date, archived, id = Random.rand(1..1000))
    super(publish_date, archived, id)
    @on_spotify = on_spotify
  end
```